examples/tutorials/Going_Deeper_on_Molecular_Featurizations.ipynb

```diff
@@ -1,6 +1,7 @@
 {
```

Comment:
<p>Let's move the exhaustive list to the bottom of the tutorial</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3862/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>